spec/helpers/mb_form_builder_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe MbFormBuilder do
```

Comment:
feels really really nice that we have unit tests for this now -- should ping our friends at GCF @bensheldon

Reply:
That is lovely 🌷